### PR TITLE
add board id for Oak Ridge National Lab Nightcrawler

### PR DESCRIPTION
**Pull request by climr**

### CODE REVIEW

File: board_types.txt [hunk 1]
```diff
@@ -62,6 +62,7 @@ AP_HW_MRO_CONTROL_ZERO_OEM            142
 AP_HW_MATEKF765_WING                  143
 AP_HW_JDMINIF405                      144
 AP_HW_KAKUTEF7_MINI                   145
+AP_HW_NIGHTCRAWLER                    146
```
Comment (peterbarker) [inline board_types.txt:65]: According to the comment on line 33, this should really be at a high number, above 1000.

Is it going to be a real pain from your end to change things?

**Comment by tridge** [comment]
We'd prefer you use an ID between 1000 and 1999, to prevent conflicts with the px4 bootloader
if you can't do that or changing from 146 is painful then let us know and we'll make an exception

**Comment by tridge** [comment]
btw, any information you can share on the nightcrawler? Sounds intriguing :-)

**Comment by climr** [comment]
no problem at all, I will make that change and request again

**Pull request closed without merging**